File: src/main/java/com/claudio_personagens/bookingcharactersapi/repositories/BookingRepository.java
package com.claudio_personagens.bookingcharactersapi.repositories;

import com.claudio_personagens.bookingcharactersapi.base.AbstractRepository;
import com.claudio_personagens.bookingcharactersapi.domain.Booking;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;

public interface BookingRepository extends AbstractRepository<Booking, String>, JpaSpecificationExecutor<Booking> {
}

File: src/main/java/com/claudio_personagens/bookingcharactersapi/repositories/BookingRepository.kt
package com.claudio_personagens.bookingcharactersapi.repositories

import com.claudio_personagens.bookingcharactersapi.base.BaseRepository
import com.claudio_personagens.bookingcharactersapi.domain.Booking
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface BookingRepository : BaseRepository<Booking, UUID>